### DISCUSSION TITLE
The latest update has broken this mod.

### DISCUSSION THREAD

**Discussion by DeenBeen** [TitanNano/TS4ControlAnySim#62] [Q&A] [answered]
Just incase you didn't know!....Also i live for this mod!

**Comment by TitanNano** [comment, marked as answer]
How is it broken? What concrete issue are you facing?

Also, please provide the debug.log file and any exceptions that you are experiencing.

**Comment by Ori28Gorg** [comment]
Hi. I was wondering if this mod was compatible with the infant update? Thanks.

**Reply by TitanNano** [reply]
I'm not aware of any concrete breakage. If you encounter issues, please let me know.

**Comment by azoresman** [comment]
Hi @TitanNano 
Here are the files you've been asking for.
I believe the errors might be related to the new "pseudo-roommates" situation called Stay over!
Hope this helps
[lastException.txt](https://github.com/TitanNano/TS4ControlAnySim/files/11425246/lastException.txt)
[debug.log](https://github.com/TitanNano/TS4ControlAnySim/files/11425247/debug.log)

**Reply by TitanNano** [reply]
The log file does not indicate that anything is going wrong inside the mod. The last exception you posted is highly unlikely to be related to this mod either. What you are seeing is most probably not caused by this mod.
